docs/src/main/paradox/consumer-metadata.md

```diff
@@ -0,0 +1,30 @@
+# Consumer Metadata
+
+To access the Kafka consumer metadata you need to create the `KafkaConsumerActor` as described in the @ref[Consumer documentation](consumer.md#sharing-the-kafkaconsumer-instance) and send messages from `KafkaConsumerActor.Metadata` to it.
+
+The metadata the Kafka Consumer provides is documented in the @javadoc[Kafka Consumer API](org.apache.kafka.clients.consumer.KafkaConsumer).
+
+The supported metadata are
+
+| Request | Reply | 
+|---------|-------|
+| ListTopics | Topics | 
```

Comment:
Maybe change to `GetTopics` as all of the other messages start with `Get`

Reply:
Note that the names were based on the KafkaConsumer methods that are actually exposed here:
https://github.com/apache/kafka/blob/837f31dd1850b179918f83338b4b4487486b2c58/clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java#L1780

(In fact the `Get` prefix was prepended to the original methods that did not have a verb in the name to make more sense as names for actor messages.)

Reply:
I see. Then current names are fine.